api/v1beta1/ovncontroller_types.go

```diff
@@ -105,6 +105,8 @@ type OVNControllerStatus struct {
 
 	// NetworkAttachments status of the deployment pods
 	NetworkAttachments map[string][]string `json:"networkAttachments,omitempty"`
+
+	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
```

Comment:
The "Observed" prefix seems to suggest a specific meaning, but I'm not sure I understand. Would "ReconciledGeneration" be closer to what we're trying to convey?

Also, do you really want omitempty?

Reply:
This is modelled after the ObservedGeneration pattern in k8s. (so I reused the name here). We can name it whatever we want I guess but it made sense to keep it similar.

Observed does mean Reconciled.

https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/deployment/deployment_controller.go#L616

Reply:
re. omitempty, I guess if it hasn't reconciled it just won't show up in the status which seemed correct to me.

Reply:
Ack (to both responses). If ObservedGeneration is a k8s thing then it makes sense to follow.